.github/workflows/macOS-CMake-toolchain.yml

```diff
@@ -0,0 +1,88 @@
+name: macOS (CMake toolchain)
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    env:
+      # Hardcoded date selecting the latest toolchain from https://swift.org/download/#snapshots.
+      toolchain-date: 2021-01-04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set environment variables
+        run: |
+          TOOLCHAIN_DATE=${{ env.toolchain-date }}
+          # TOOLCHAIN_SHORT_DATE: TOOLCHAIN_DATE with hypens removed. Used for toolchain identifier and version.
+          TOOLCHAIN_SHORT_DATE="${TOOLCHAIN_DATE//-/}"
+          TOOLCHAIN_NAME=swift-DEVELOPMENT-SNAPSHOT-${TOOLCHAIN_DATE}-a
+          TF_TOOLCHAIN_NAME=swift-tensorflow-DEVELOPMENT-SNAPSHOT-${TOOLCHAIN_DATE}-a
+          echo "TOOLCHAIN_DATE=${TOOLCHAIN_DATE}" >> $GITHUB_ENV
+          echo "TOOLCHAIN_SHORT_DATE=${TOOLCHAIN_SHORT_DATE}" >> $GITHUB_ENV
+          echo "TOOLCHAIN_NAME=${TOOLCHAIN_NAME}" >> $GITHUB_ENV
+          echo "TF_TOOLCHAIN_NAME=${TF_TOOLCHAIN_NAME}" >> $GITHUB_ENV
+          echo "SWIFT_APIS_BUILD_DIR=/tmp/BinaryCache/tensorflow-swift-apis" >> $GITHUB_ENV
```

Comment:
Not sure how much value this has to be honest.

Reply:
`$SWIFT_APIS_BUILD_DIR` is used twice (in the CMake build and install steps), so I think it does add some value as a variable. For example, if `macOS-CMake-toolchain.yml` is cloned for a different platform, it would only be necessary to update `SWIFT_APIS_BUILD_DIR` instead of its two use sites.

I'll proceed as is if that is okay.

Reply:
The single variable isn't particularly useful to hoist IMO.  It is just the location of the build tree.  You could also just replace that with `build` and use a relative path :)

Reply:
That's fair, replaced with `build` :)